src/Server.jl

```diff
@@ -737,8 +756,8 @@ function make_pull_request(pp::ProcessedParams, rp::RequestParams, rbrn::RegBran
     ref = get_html_url(rp.evt.payload)
 
     # FYI: TagBot (github.com/apps/julia-tagbot) depends on the "Repository", "Version",
-    # and "Commit" fields. If you're going to change the format here, please ping
-    # @christopher-dG and make sure that WebUI.jl has also been updated.
+    # "Commit", and "Patch notes" fields. If you're going to change the format here,
+    # please ping @christopher-dG and make sure that WebUI.jl has also been updated.
     params["body"] = """
```

Comment:
We should factor this out and make it a common method called by WebUI and the comment bot.